docs/manual/backend/backend-keyboard-shortcuts.en.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Keyboard shortcuts"
+description: "To speed up the workflow when working with Contao, there are several keyboard shortcuts in the backend, 
+with which certain commands can be called directly."
+url: "administration-area/keyboard-shortcuts"
```

Comment:
Should we really also change the URL for the English version? Default would be the file path. I thought we would be using that.

Reply:
I would go with «administration-area/backend-keyboard-shortcuts». Why change the wording? [Past docs](https://docs.contao.org/books/manual/3.4/en/02-administration-area/index.html).

Reply:
Ah I see. Hm, though we do write "back end" everywhere else (e.g. in the dev docs). Do you think we should change it to "back end" nevertheless?